### PR TITLE
close #7 tab bar config

### LINKED ISSUE #7: Tab Bar Config

### PR DESCRIPTION
**Pull request by darot-chen**

### CODE REVIEW

**Comment by theachoem** [comment]
@darot-chen please rebase with develop branch, you have 2 commit

File: lib/storage/bool_preference_storage.dart [hunk 1]
```diff
@@ -9,17 +9,17 @@ abstract class BoolPreference {
     await _instance.setBool(key, !current);
   }
 
-  Future<void> setBool({bool value}) async {
+  Future<void> setBool({bool? value}) async {
```
Comment (theachoem) [inline lib/storage/bool_preference_storage.dart]: we don't expect to store null in share preference. If it is a null value, we remove it directly using method `clear()`

result should be done by remove that ? from bool, `setBool({required bool value}){}`

**Comment by darot-chen** [comment]
Dear Mr.@theacheng  

I have fixed the problem. 

Please review and merge the code. If you have any question or problem please mention me in the comment. 

Thanks you. Kiss

**Comment by theachoem** [comment]
@darot-chen Well done brother, thank you!

**Pull request merged**